client/src/pages/HomePage/sections/HomeHeader/HomeHeader.style.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+export const StyledHeader = styled.header`
+ background-image: url(https://user-images.githubusercontent.com/28222381/45626954-72961080-ba99-11e8-9215-538504f985ea.jpg);
+ background-repeat: no-repeat;
+ background-size: cover;
+ height: 250px;
+ background-position: bottom;
```

Comment:
Since I know you love a one liner, the background CSS on lines 4,5,6, and 8 could be done in one line using https://www.w3schools.com/cssref/css3_pr_background.asp.

Reply:
Thanks, done

Reply:
Didn't work for some reason, reverted :)